app/utils/answersHelpers/consignment/HouseConsignmentAnswersHelper.scala

```diff
@@ -82,15 +83,16 @@ class HouseConsignmentAnswersHelper(
   def itemSections: Seq[Section] =
     userAnswers.get(ItemsSection(houseConsignmentIndex)).mapWithIndex {
       case (_, itemIndex) =>
-        val helper = new ConsignmentItemAnswersHelper(userAnswers, houseConsignmentIndex, itemIndex)
+        val helper        = new ConsignmentItemAnswersHelper(userAnswers, houseConsignmentIndex, itemIndex)
+        val packageHelper = new PackagingAnswersHelper(userAnswers, houseConsignmentIndex, itemIndex)
         AccordionSection(
           sectionTitle = Some(messages("unloadingFindings.subsections.item", itemIndex.display)),
           rows = Seq(
             helper.descriptionRow,
             helper.grossWeightRow,
             helper.netWeightRow
           ).flatten,
-          children = helper.packageSections ++ Seq(
+          children = packageHelper.packageSections ++ Seq(
```

Comment:
`packageSections` can remain in ConsignmentItemAnswersHelper as that is the level it lives at